File: examples/fib.py
#from __builtin__ import raw_input, int
from time import clock as fclock

def fib(n):
	'''fib computes the nth number of the fibonacci sequence'''
	if n == 1:
		return 1
	elif n == 2:
		return 1

	return fib(n - 1) + fib(n - 2)

def m(maxfib):
	start = fclock()
	sum = 0
	#for i in range(1, 35):
	#    sum += fib(i)
	i = 1
	while i < maxfib:
		sum += fib(i)
		i += 1
	end = fclock()
	print sum, ', found in', end - start, 'seconds'

#maxfib = int(raw_input('sum fib to '))	
m(35)#(maxfib)

File: examples/fib.cpp

#import <iostream>
#import <time.h>

int fib(int n) {
    if (n == 1) {
        return 1;
    } else if (n == 2) {
        return 1;
    } else {
        return fib(n - 1) + fib(n - 2);
    }
}

int main(int argc, char **argv) {
    int start = clock();
    
    long sum = 0;
    for (int i = 1; i < 35; i++) {
        sum += fib(i);
    }
    
    int end = clock();
    float seconds = (float)(end - start) / CLOCKS_PER_SEC;
    std::cout << sum << ", found in " << seconds << " seconds\n";
}
